src/Button/Button.test.jsx

```diff
@@ -18,6 +18,18 @@ describe('<Button />', () => {
       const tooltip = wrapper.find('.btn');
       expect(tooltip.hasClass('btn-brand')).toEqual(true);
     });
+    it('renders with props iconBefore and size', () => {
+      const tree = renderer.create((
+        <Button iconBefore={Close} size="xs">Button</Button>
```

Comment:
[curious] Is this `size="xs"` throwing a prop type warning when running these tests given that `Button` component only accepts `size={sm|md|lg}`. Granted, if we pass-through `size="xs"`, it'll change the icon size via `.pgn__icon__xs` but may want to only assert on `size` options that is supported by `Button` here.